cli/cmd/plugin/unmanaged-cluster/cmd/create.go

```diff
@@ -59,8 +62,9 @@ Exit codes are provided to enhance the automation of bootstrapping and are defin
 9  - Could not install kapp controller to cluster.
 10 - Could not install core package repo to cluster.
 11 - Could not install additional package repo
-12 - Could not install CNI package.
-13 - Failed to merge kubeconfig and set context`
+12 - Could not install designated profile
+13 - Could not install CNI package.
```

Comment:
For backwards compat, can we keep this one as `12` and add all the new ones above `12`?